src/main.c

```diff
@@ -374,27 +377,35 @@ main(void)
   // ensure exception will work as planned
   os_boot();
 
-  UX_INIT();
+  for (;;) {
+    UX_INIT();
 
-#if defined(TARGET_NANOS)
-  UX_MENU_INIT();
-#endif
+  #if defined(TARGET_NANOS)
```

Comment:
I think it's more idiomatic to not indent preprocessor macros, and the halfway indent here just seems odd regardless.

Reply:
sure -- removed the indentation :+1: